### ISSUE TITLE
Add Integeration Test Bot job to BigchainDB production deployment template

### ISSUE THREAD

**Issue by shahbazn** [bigchaindb/bigchaindb#1920]
### Description
We run a very simple integeration test-bot which daily posts some transactions on test net and prod net to provide test coverage for POST workflows which are not covered by Runscope. Its run as a CRON job on a separate VM which require it to be updated when ever we do a new test net/prod net deployment but since its not part of the https://docs.bigchaindb.com/projects/server/en/latest/production-deployment-template/index.html it usually gets missed. So we need to add a kubernetes https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/ as part of our deployment template so that a test-bot gets deployed with each new test/prod net deployment

**Comment by ttmc** [comment]
Will this test-bot be running in the same Kubernetes cluster as a BigchainDB node? That seems a bit risky. If the Kubernetes cluster fails completely, then the test-bot will also stop testing. I think the test-bot should be on a stand-alone VM.

**Comment by shahbazn** [comment]
@ttmc yes it will be on the same cluster but different namespace as the bdb node. These tests run on nightly bases and we have other tests which run more frequently and if cluster fails we get alarms on those tests as well. Infra can fail anywhere k8s cluster or VM. Ideally cluster health should be monitored separately with a monitoring system and I thinks it should be fine to run the test in the same cluster. Let me know what you think

**Comment by ttmc** [comment]
> Ideally cluster health should be monitored separately with a monitoring system...

That makes sense.

**Comment by ttmc** [comment]
@shahbazn Is there a PR in this repository that adds the test-bot (dockerized in https://github.com/ascribe/ipdb-tests/pull/4 ) to the production deployment template?
